maintenance-mode.php

```diff
@@ -44,3 +56,20 @@ function vip_maintenance_mode_load_plugin_textdomain() {
 	load_plugin_textdomain( 'maintenance-mode', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
 }
 add_action( 'init', 'vip_maintenance_mode_load_plugin_textdomain' );
+
+/**
+ * Displays a warning in the admin to inform users about the VIP_MAINTENANCE_MODE constant
+ *
+ * Only users who can activate plugins will see the warning.
+ *
+ * @since 0.1.1
+ */
+function vip_maintenance_mode_admin_notice__constant_not_set() {
+	if ( !current_user_can( 'activate_plugins' ) ) {
+		return;
+	}
+
+	$class = 'notice notice-warning';
+	$message = __( 'Maintenance Mode won\'t work until you set the VIP_MAINTENANCE_MODE constant to <code>true</code>.', 'maintenance-mode' );
+	printf( '<div class="%1$s"><p>%2$s</p></div>', $class, wp_kses( $message, array( 'code' => array() ), array() ) );
```

Comment:
At the moment, the value of `$class`is safe because it is hardcoded. Is it required to escape hardcoded values? Or considered as a best practice to be safe the day this value is no more hardcoded? I'm curious.